### DISCUSSION TITLE
[feature idea] cli: install package without any dependencies

### DISCUSSION THREAD

**Discussion by redbrain** [npm/rfcs#262] [RRFC] [answered]
I'd like to be able to install an npm package without any of its dependencies, including direct dependencies.

While this would normally break functionality of packages, this would be useful for bug-fixing, developing, or (in my case) if you use custom-patched dependencies that would be overwritten on the install of a package that requires its original version.

I'm not sure if this is a place to ask for features to be implemented, but I'd like to see an easy way to do this (i.e. a cli flag `--no-dependencies` with plenty of warnings around it) in a future version of npm.

**Comment by ruyadorno** [comment, marked as answer]
hi @redbrain I do agree that's an useful workflow and while it's not directly in the npm cli today, it's possible to benefit from that functionality today using [pacote](https://github.com/npm/pacote) (part of the npm package ecosystem).

Quick example, extract the contents of the latest published version of `react` to a `<path>` in your file system:

```
$ npx pacote extract react@latest ./react
```

You can also learn more about the pacote cli usage running: `npx pacote --help`

**Reply by redbrain** [reply]
Thanks, this is perfect!

**Reply by redbrain** [reply]
Is there a way to mark the discussion as closed/solved?

**Reply by ruyadorno** [reply]
done! 😄
